RevenueCatUI/Modifiers/ViewExtensions.swift

```diff
@@ -109,7 +109,7 @@ extension View {
     @ViewBuilder
     func scrollableIfNecessaryWhenAvailable(_ axis: Axis = .vertical, enabled: Bool = true) -> some View {
         if enabled {
-            if #available(iOS 16.0, macOS 13.0, tvOS 16.0, watchOS 9.0, *) {
+            if #available(iOS 16.0, macCatalyst 16.6, macOS 13.0, tvOS 16.0, watchOS 9.0, *) {
```

Comment:
Any reason for using `16.6` and not `16.0` ?

Reply:
sorry, merged & then noticed your comment.
It's the same as in https://github.com/LinearityGmbH/purchases-ios/pull/9/files

Reply:
Ok, I see. Was there any reason for the other one ? Because `ViewThatFits` is available starting 16.0